app/views/home/index.slim

```diff
@@ -6,7 +6,10 @@
         | #pivorak
   .pk-home-layout__content-wrapper
     .pk-home-layout__content
-      p = t('coming_soon.next_event', upcoming_date: upcoming_date)
+      - if planned_event.present?
```

Comment:
What about helper like this?
```
def comming_soon_message
  return t('coming_soon.getting_ready') if planned_event.blank?

  t('coming_soon.next_event', upcoming_date: upcoming_date)
end
```

After that just use `comming_soon_message` in `.slim`
It helps if logic will be changed later.

Reply:
Happy to add this, one question: you prefer I add this as a helper method inside a controller, or inside a regular helper module? I am not used to having so many helper methods inside the controller, and they seem to be used a lot in the project.

Reply:
@sergiy-koshoviy Extracted to a helper method inside the controller, as it seems to be the convention in the project.

Reply:
In general in RoR style guides we have use regular helper module. We can create helper inside in controller if we need method in the controller and in the view for example.